starlite/plugins/sql_alchemy.py

```diff
@@ -147,7 +147,7 @@ def providers_map(self) -> Dict[Type[TypeEngine], Callable[[Union[TypeEngine, Ty
             mssql.DATETIMEOFFSET: lambda x: datetime,
             mssql.IMAGE: self.handle_string_type,
             mssql.MONEY: lambda x: Decimal,
-            mssql.NTEXT: self.handle_string_type,
+            mssql.NEXT: self.handle_string_type,
```

Comment:
A bit off the reservation here. Have you absolutely verified this?

Reply:
I'll respond to this one first.  It is certainly off topic, and wasn't supposed to be included.  I'll remove this edit from the PR.